Unity-Technologies/ScriptableRenderPipeline
# [Universal RP] bugfix 1175396

### Purpose of this PR
Animated materials(shaders using `_Time`) stopped working in scene view reliably after a previous bug fix. Now the editor has a different codepath in which is uses `Time.realtimeSinceStartup` which increments when not in playmode and also respects the Animated Materials checkbox in the scene view.

![AnimatedMatFixPR](https://user-images.githubusercontent.com/9811576/63091922-46f90000-bf60-11e9-9194-833a14ee9806.gif)

---
### Release Notes
Animated materials now work in, and respect the toggle in the scene view.

---
### Testing status
**Katana Tests**: Running

**Manual Tests**: What did you do?
- [N/A] Checked new resources path for the reloader (in devloper mode, you have a button at end of resources that check the pathes)
- Other: 

**Automated Tests**: N/A we don't have the ability right now to capture scene view output.

Launch Katana (Link on master here - update for your branch):
https://katana.bf.unity3d.com/projects/com.unity.render-pipelines/builders?ScriptableRenderLoop_branch=universal%2Fbugfix%2F1175396&automation-tools_branch=master&unity_branch=trunk&sort=1-asc

Alternative, launch Yamato (Select your branch):
https://yamato.prd.cds.internal.unity3d.com/jobs/78-ScriptableRenderPipeline/tree/universal%252Fbugfix%252F1175396/.yamato%252Fupm-ci-universal.yml%2523All_Universal/345715/job

Any test projects to go with this to help reviewers?
N/A

---
### Overall Product Risks
**Technical Risk**: None

**Halo Effect**:  Low - Editor only and edit mode only, playmode is the same as before

---
### Comments to reviewers
Notes for the reviewers you have assigned.
